tools/mason/MasonSearch.chpl

```diff
@@ -122,6 +119,25 @@ proc masonSearch(ref args: list(string)) {
   }
 }
 
+
+/* Sort the results in a order such that results that startWith needle
+   are displayed first */
+proc rankResults(results, query) {
+  use Sort;
+  record Comparator { }
+  proc Comparator.compare(a, b) {
+    if a.toLower().startsWith(query) && !b.toLower().startsWith(query) then return -1;
```

Comment:
I suspect there will still be a difference for MacOS vs. Linux ordering for lower/uppercase package names that get sorted with this comparator, e.g. `foo` and `FooBar` searched with `mason search fo`

This isn't a big deal IMO, but we should be aware of it in the future when adding mason search tests.

Reply:
I agree !